Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/J2ObjCParser.kt

```diff
@@ -0,0 +1,218 @@
+package org.jetbrains.kotlin.native.interop.indexer
+
+import org.jetbrains.org.objectweb.asm.ClassReader
+import org.jetbrains.org.objectweb.asm.ClassVisitor
+import org.jetbrains.org.objectweb.asm.MethodVisitor
+import org.jetbrains.org.objectweb.asm.Opcodes
+import org.jetbrains.org.objectweb.asm.Type.getArgumentTypes
+import org.jetbrains.org.objectweb.asm.Type.getReturnType
+import java.util.jar.JarFile
+
+/**
+ * Visits a Java class and builds an ObjCClass
+ */
+class J2ObjCParser: ClassVisitor(Opcodes.ASM5) {
+
+  var className:String = ""
+  val methodDescriptors = mutableListOf<Triple<String, String, Int>>()
```

Comment:
Should I make a special type for this rather than leaving it as a Triple?
I wanted to keep together the information needed to build a method (name, descriptorstring, accesslevel)

Reply:
since this appears to be a public property, a specialized type may be nice.  I'd recommend a data class, with the added benefit that it supports destructuring just like Triple.

Reply:
Done.